pkg/webui/lib/errors/utils.js

```diff
@@ -349,24 +376,34 @@ export const toMessageProps = error => {
 
 /**
  * `ingestError` provides a unified error ingestion handler, which manages
- * forwarding to sentry and other logic that should be applied when errors
+ * forwarding to Sentry and other logic that should be applied when errors
  * occur. The error object is not modified.
  *
  * @param {object} error - The error object.
  * @param {object} extras - Sentry extras to be sent.
  * @param {object} tags - Sentry tags to be sent.
+ * @param {Function} done - Optional callback called after execution.
```

Comment:
I dont see any usage of this callback in application code, only in tests. We shouldnt adjust function signatures just for testing purposes.